modules/ROOT/pages/managing-asset-deletion.adoc

```diff
@@ -0,0 +1,16 @@
+= Managing Asset Deletion
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Control how assets are deleted in Exchange from *Settings*. Admins can choose how asset owners can delete asset versions in Exchange. They can enable or disable the ability to permanently delete an asset to reuse the GAV (group ID, asset ID, and version) for a new asset. It doesn't matter when the asset version was created. When enabled, asset owners see an option to allow the GAV to be reused when deleting an asset version. When disabled, the GAV can't be reused after deleting the asset version.
```

Comment:
What about?
[...] They can enable or disable the ability `in order to` permanently delete an asset to reuse the GAV [...]